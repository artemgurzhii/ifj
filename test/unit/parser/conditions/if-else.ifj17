if a then
dim a as string
else
dim a as string
end
