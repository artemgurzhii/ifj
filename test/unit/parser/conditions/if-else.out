(if (id a)
  (dim
    (decl
      (id a)
      : (id string))))
(else
  (dim
    (decl
      (id a)
      : (id string))))
